skbio/sequence/_grammared_sequence.py

```diff
@@ -277,9 +279,9 @@ def degenerate_chars(cls):
         """
         return set(cls.degenerate_map)
 
-    @abstractproperty
     @classproperty
-    @stable(as_of='0.4.0')
+    @deprecated(as_of='0.5.0', until='0.5.2',
```

Comment:
Can you change `as_of` to `0.4.2-dev`? That way if 0.5.0 isn't the next release we'll be able to catch and update the `as_of` version.